Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -0,0 +1,175 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class JumpBackInTests: BaseTestCase {
+    func closeKeyboard() {
+        waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
+        navigator.performAction(Action.CloseURLBarOpen)
+        navigator.nowAt(NewTabScreen)
+    }
+
+    override func setUp() {
+        super.setUp()
+
+        closeKeyboard()
+
+        // Since I can't scroll on M1, closing the "Switch Your Default Browser"
+        // box makes the "Jump Back In" section visible without scrolling in the
+        // last part of the test.
+        if app.staticTexts["Switch Your Default Browser"].exists {
```

Comment:
"Switch Your Default Browser" takes up some room that I may need to `swipe` in the test. 😞